tests/by-util/test_whoami.rs

```diff
@@ -1,63 +1,47 @@
-use crate::common::util::*;
+//  * This file is part of the uutils coreutils package.
+//  *
+//  * For the full copyright and license information, please view the LICENSE
+//  * file that was distributed with this source code.
 
-// Apparently some CI environments have configuration issues, e.g. with 'whoami' and 'id'.
-// If we are running inside the CI and "needle" is in "stderr" skipping this test is
-// considered okay. If we are not inside the CI this calls assert!(result.success).
-//
-// From the Logs: "Build (ubuntu-18.04, x86_64-unknown-linux-gnu, feat_os_unix, use-cross)"
-// stderr: "whoami: failed to get username"
-// Maybe: "adduser --uid 1001 username" can put things right?
-fn skipping_test_is_okay(result: &CmdResult, needle: &str) -> bool {
-    if !result.succeeded() {
-        println!("result.stdout = {}", result.stdout_str());
-        println!("result.stderr = {}", result.stderr_str());
-        if is_ci() && result.stderr_str().contains(needle) {
-            println!("test skipped:");
-            return true;
-        } else {
-            result.success();
-        }
-    }
-    false
-}
+use crate::common::util::*;
```

Comment:
could you please fix that warning? thanks

Reply:
done